### PR TITLE
Creating multi-architecture images using docker manifest tool

### PR DESCRIPTION
**Pull request by damora**
Added platform specific Dockerfiles for ppc64le, amd64 and ubuntu and centos
Modified existing Makefile adding a manifest build section:
1. This section pushes the architecture specific images to ppc64le-kube-arbitrator and amd64-kube-arbitrator repositories
2. Creates the manifest that includes
   damora/ppc64le-kube-arbitrator:ubuntu-0.1
    damora/ppc64le-kube-arbitrator:centos-0.1
   damora/amd64-kube-arbitrator:ubuntu-0.1
3. pushes the manifest to damora/kube-arbitrator

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 BIN_DIR=_output/bin
+DOCKER_REPO=damora
```
Comment (k82cn) [inline Makefile:2]: please use kubearbitrator

File: deployment/Dockerfile.ppc64le.centos [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM  ppc64le/centos:7
```
Comment (k82cn) [inline deployment/Dockerfile.ppc64le.centos:1]: Is it necessary to provide two image?

**Comment by k82cn** [comment]
please sign CNCF-CLA :)

File: deployment/Dockerfile.ppc64le.centos (hunk 2 above)
Comment (damora) [reply deployment/Dockerfile.ppc64le.centos:1]: you mean an Ubuntu and Centos image? Probably not.

File: Makefile (hunk 1 above)
Comment (k82cn) [reply Makefile:2]: oh, I create kubsigs for us.

**Comment by jeefy** [comment]
I don't think this will work. 

You have to have an Edge build of Docker and enable experimental-CLI features to be able to use the manifest command. Travis doesn't have this version of Docker by default.

See https://github.com/kubernetes/dashboard/pull/3143/files for an example :)

**Pull request closed without merging**